src/reduxTokenAuthConfig.js

```diff
@@ -0,0 +1,26 @@
+import { generateAuthActions } from 'redux-token-auth'
+
+const config = {
+  authUrl: "http://localhost:3002/api/v1/auth",
```

Comment:
do here the same we do in the App.js with the `loc` and `her` variables. and use single quotes.

Reply:
Fixed!